### ISSUE TITLE
[1.16.4] Cannot claim chunks in The End

### ISSUE THREAD

**Issue by NillerMedDild** [FTBTeam/FTB-Chunks#82]
Claiming Chunks in the End isnt possible, every time I try claiming a chunk in the menu nothing happens.

### Versions
- Forge-1.16.4-35.1.29
- ftb-chunks-1604.2.2.54

Original issue: https://github.com/NillerMedDild/Enigmatica6/issues/704

**Comment by Letoric** [comment]
That's not a  bug it's a config option, and also really you shouldn't do it because the number of entities would be really  bad (all the endermen)

**Comment by NillerMedDild** [comment]
Endermen can spawn without players nearby?

**Comment by Danekjovax** [comment]
I am trying to find this option.  Which file/option am I to look for in config?

**Comment by exline22** [comment]
> I am trying to find this option. Which file/option am I to look for in config?

In single player ATM6, I found it under the saves folder, then "world name" folder, serverconfig folder, then there is a ftbchunks-server.toml.  I edited it by removing the minecraft:theend from between the quotes; when I tried just commenting out the line, the game detected the line as missing and rewrote it to the file!

Can confirm it worked with Void Miner in the End.
